### PR TITLE
`iTwinUI-icons-elements` package version bump

### PR DESCRIPTION
**Pull request by FlyersPh9**
## Changes

- Bump `@itwin/itwinui-icons-elements` from `0.2.0` to `0.20.0`, needed for #1676.
- Header, file upload, tile, and toast required a CSS fix to show the correct color.  Prior to these fixes the inline styling on the SVG was taking preference (`fill: var(--iui-color-icon-muted, currentColor)`).
- Select demo page's disabled examples did not have all the same class names as the other examples, resulting in a non-1:1 ratio icon.

## Testing

- Ran visual test and visually confirmed each one.  The only test image I am unsure why it changed was the side navigation.

## Docs

N/A

### CODE REVIEW

**Review by rohan-kadkol — APPROVED** [review]

File: packages/itwinui-css/src/toast/toast.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 /// category must be one of: informational, positive, warning, negative
 @mixin iui-toast-category($category) {
+  --_iui-toast-icon-color: var(--iui-color-icon-#{$category});
```
[the remark this answers is not in the document]
Comment (rohan-kadkol) [reply packages/itwinui-css/src/toast/toast.scss:8]: Sounds good, undid (https://github.com/iTwin/iTwinUI/pull/1732/commits/ac2c28d6f11ab7defbce1a6b70019e9aa6e3bcb6)

File: packages/itwinui-css/src/header/header-buttons.scss [hunk 2]
```diff
@@ -223,7 +227,10 @@ $iui-side-navigation-icon-margins: calc(1.5 * var(--iui-size-m));
     font-size: var(--iui-font-size-0);
     color: var(--iui-color-text-muted);
     @media (prefers-reduced-motion: no-preference) {
-      transition: visibility 0s linear 0s, width var(--iui-duration-1) ease-out, height var(--iui-duration-1) ease-out,
+      transition:
```
[the remark this answers is not in the document]
Comment (rohan-kadkol) [reply packages/itwinui-css/src/header/header-buttons.scss:230]: When I undid this change and reformatted the file again, I noticed it went back to this changed state. So, I guess we can keep this formatting change?

File: packages/itwinui-css/src/toast/toast.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (mayank99) [reply packages/itwinui-css/src/toast/toast.scss:8]: remove the variable from line 8 too

File: packages/itwinui-css/src/file-upload/file-upload.scss [hunk 3]
```diff
@@ -31,7 +31,8 @@
     }
 
     > svg,
-    .iui-template-icon {
+    .iui-template-icon,
+    .iui-template-icon > svg {
```
[the remark this answers is not in the document]
Comment (rohan-kadkol) [reply packages/itwinui-css/src/file-upload/file-upload.scss]: When using `:is(> svg, .iui-template-icon, .iui-template-icon > svg)`, I noticed that the generated CSS had some issues. When I added an & before the first sub-selector in `:is()`, the generated CSS included the selector correctly.

```diff
-    :is(> svg, .iui-template-icon, .iui-template-icon > svg) {
+    :is(& > svg, .iui-template-icon, .iui-template-icon > svg) {
```

However, when I tried to commit this change, I got the following error:

Screenshot 2023-12-12 at 5 27 20 PM

Thus, I went with the below version with `> svg` out of the `:is()` (https://github.com/iTwin/iTwinUI/pull/1732/commits/e241b81fe32f383200f50ecec636566d342c59f1):

```diff
-    :is(> svg, .iui-template-icon, .iui-template-icon > svg) {
+    > svg, :is(.iui-template-icon, .iui-template-icon > svg) {
```

File: packages/itwinui-css/src/toast/toast.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (rohan-kadkol) [reply packages/itwinui-css/src/toast/toast.scss]: Oops, removed (https://github.com/iTwin/iTwinUI/pull/1732/commits/2cee1de544d5dd1b733ac02990b174d2ceaa8d8b)

**Review by mayank99 — APPROVED** [review]

**Pull request merged**